src/style.css

```diff
@@ -0,0 +1,114 @@
+@font-face {
+  font-family: "Avenir", sans-serif;
+  src: url(./fonts/Avenir.ttc);
+  font-style: normal;
+  font-weight: 300;
+}
```

Comment:
I see you use 900 weight as well, so set up font face for all of the weights

Reply:
done